src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -871,6 +873,9 @@ def forward(
         hidden_states = transformer_outputs[0]
 
         lm_logits = self.lm_head(hidden_states)
+        # the original softmax is computed in fp32 during training
+        # https://github.com/bigscience-workshop/Megatron-DeepSpeed/blob/09a35f53abac96903fee50787426b7ee5f63fc62/megatron/model/gpt_model.py#L289-L29
+        # to replicate the same behavior as training, make sure to cast lm_logits to fp32
```

Comment:
You could add the line of code to uncomment as well here.

Reply:
do you think we still need this comment section if we put it in the tips section of model card?

Reply:
Not necessarily!